presentation/src/main/java/me/androidbox/presentation/agenda/screen/AgendaScreen.kt

```diff
@@ -124,7 +126,27 @@ fun AgendaScreen(
                         isAgendaCompleted = false
                     ) {
                         println("Event ${agendaItem.id} has been clicked")
+                        agendaScreenEvent(AgendaScreenEvent.OnAgendaItemClicked(agendaItem))
+                        agendaScreenEvent(AgendaScreenEvent.OnChangeShowEditAgendaItemDropdownStatus(shouldOpen = true))
                     }
+
+                    AgendaDropDownMenu(
+                        modifier = Modifier
+                            .background(color = MaterialTheme.colorScheme.dropDownMenuBackgroundColor)
+                            .align(Alignment.BottomEnd),
+                        shouldOpenDropdown = agendaScreenState.shouldOpenEditAgendaDropdown,
+                        onCloseDropdown = {
+                            agendaScreenEvent(
+                                AgendaScreenEvent.OnChangeShowEditAgendaItemDropdownStatus(shouldOpen = false))
+                        },
+                        listOfMenuItemId = listOf(R.string.open, R.string.edit, R.string.delete),
```

Comment:
Didn't you create an AgendaMenuActionType for this?
You can use those, and have the Enum receive the ids as params

So smth like:

```
enum class AgendaMenuActionType(@StringRes val id: Int) {
    OPEN(R.string.open),
    EDIT(R.string.edit),
    DELETE(R.string.delete)
}

And then use it like:

AgendaMenuActionType.values() to get a list of AgendaMenuActionType

Reply:
@MKiperszmid I will update